lib/pages/add_bridge/add_bridge.dart

```diff
@@ -628,6 +799,90 @@ class BotController extends State<AddBridge> {
   // 📌 ************************** Messenger & Instagram **************************
   // 📌 ***********************************************************************
 
+  Future<void> bridgeLoginApi(
+      BuildContext context,
+      WebviewCookieManager cookieManager,
+      ConnectionStateModel connectionState,
+      SocialNetwork network,
+      ) async {
+    final flowID = network.flowId;
```

Comment:
the supported flows could be fetched from the API. We should discuss how we choose to do it